swagger.yaml

```diff
@@ -39,23 +127,160 @@ parameters:
     description: If true, JSON response will be pretty-printed
 
 definitions:
-  Info:
+  RoutesResultObject:
     properties:
-      name:
+      data:
+        type: array
+        items:
+          $ref: "#/definitions/RouteResourceObject"
+  RouteResultObject:
+    properties:
+      data:
+        $ref: "#/definitions/RouteResourceObject"
+  RouteResourceObject:
+    properties:
+      id:
         type: string
-        description: Name of the API.
-      time:
+        description: routeID
+      type:
         type: string
-        description: Build time in ISO 8601 format.
-      unixTime:
-        type: integer
-        description: Build time as milliseconds since the unix epoch.
-      commit:
+        description: route
+      attributes:
+        properties:
+          description:
+            type: string
+            description: Description of bus route
+          encodedPolyline:
+            type: string
+            description: Encoded polyline for mapping route using Google's mapping APIs
+          mapColor:
+            type: string
+            description: Hexadecimal color value for route line on map
+          latitude:
+            type: number
+            format: float
+            description: Default latitude to go to when selecting route on map
+          longitude:
+            type: number
+            format: float
+            description: Default longitude to go to when selecting route on map
+          zoomLevel:
+            type: integer
+            description: Default zoom level to use when selecting route on map
+          stops:
+            type: array
+            description: Locations where the bus stops along the route
+            items:
+              properties:
+                stopID:
+                  type: string
+                description:
+                  type: string
+                latitude:
+                  type: number
+                  format: float
+                  description: Latitude of stop
+                longitude:
+                  type: number
+                  format: float
+                  description: Longitude of stop
+                order:
```

Comment:
I was thinking about whether we should include this field in our API. The vendor includes this field, but should just give the order of elements in this array in the correct order? Or both?

Reply:
Hmm, i agree. It doesn't sound necessary.